### PR TITLE
Track bundle size.

### LINKED ISSUE #161: Track freedom.js size
We should add a page to our dashboard tracking final library size over time.

### PR DESCRIPTION
**Pull request by willscott**
Resolve #161

This saves to a spreadsheet that we can then build graphs off of. Tracks file size of freedom.js (non-minimized) & build time on travis.

### CODE REVIEW

File: tasks/report-stats.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  var request = require('request'),
+    fs = require('fs'),
+    start = new Date();
+
+  grunt.registerTask('reportStats', 'Report build statistics.', function () {
+    var done = this.async(),
+      size = fs.statSync('freedom.js').size;
+
+    request({
+      url: 'https://script.google.com/macros/s/' +
```
Comment (ryscheng) [inline tasks/report-stats.js]: Source code for this? Where is this visualized?

File: tasks/report-stats.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            'AKfycbyssdIXai4pthm-mikC-jrUYUjGKzVmV-NgIVilIv9dJPcvRaU/exec',
+      qs: {
+        time: new Date() - start,
+        size: size,
+        commit: process.env.TRAVIS_COMMIT,
+        branch: process.env.TRAVIS_BRANCH,
+        failures: 0,
+        freedom: process.env.STAT_KEY
```
Comment (ryscheng) [inline tasks/report-stats.js]: What defines this? Doesn't exist here:
http://docs.travis-ci.com/user/ci-environment/#CI-environment-OS

Comment (willscott) [inline tasks/report-stats.js]: Defined as a travis custom environment variable. used to make sure only travis is able to post stats.

**Comment by ryscheng** [comment]
:+1:

**Pull request merged**